### DISCUSSION TITLE
Is there a way to send sentences with furigana to anki?(mokuro)

### DISCUSSION THREAD

**Discussion by YashaNoos** [arianneorpilla/jidoujisho#264] [Q&A] [answered]
Hi, 
I was wondering if there is a way to generate furigana for the Sentence (and Cloze Before, Inside, After) and sending the result directly to anki, the same way it already work for Term and Furigana.
Right now I'm using the AJT add-on to generate furigana on my PC once the card is created but it's a bit cumbersome, I need to add 4 new field to generate the furigana and trigger the conversion manually four times for each new card.

I just discovered your app recently and it's such a wonderful tool. Thanks a lot for your work and have a nice day!

**Comment by arianneorpilla** [comment, marked as answer]
I don't plan to implement this unfortunately.

**Reply by YashaNoos** [reply]
Thank for the quick answer! I'll  try to find a way around it.
